spec/requests/workflows/workflows_skip_all_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Skip all in a workflow' do
+  let(:wf) { FactoryBot.create(:workflow_step) }
+  let(:druid) { wf.druid }
+  let(:workflow_template) { WorkflowTemplateLoader.load_as_xml('accessionWF') }
+
+  before do
+    allow(QueueService).to receive(:enqueue)
+  end
+
+  describe 'POST skip_all' do
+    let(:druid) { 'druid:bb123bb1234' }
+    let(:workflow) { 'accessionWF' }
+
+    let(:headers) { { 'Content-Type' => 'application/xml' } }
+    let(:xml_content) { '<?xml version="1.0"?><process name="skip-all" note="note"/>' }
+
+    context 'when the version is passed without context' do
```

Comment:
i'm not sure i understand the context description here, especially the "without context" part for.  wondering whether it's copypasta, or if i'm missing something.

Reply:
My bad. I copied a previous test and didn't change that context.